settings.php

```diff
@@ -934,6 +947,21 @@ function convertseconds($argument)
                                                     <a href="https://dnssec.vs.uni-due.de/" rel="noopener" target="_blank">here</a>.</p>
                                                 </div>
                                                 <br>
+                                                <h4><a id="ratelimit"></a>Rate-limiting</h4>
+                                                <p>Block clients making more than <input type="number" name="rate_limit_count" value="<?=$rate_limit_count?>" min="0" step="10" style="width: 5em;"> queries within
+                                                <input type="number" name="rate_limit_interval" value="<?=$rate_limit_interval?>" min="0" step="10" style="width: 3em;"> seconds.</p>
```

Comment:
```suggestion
                                                <input type="number" name="rate_limit_interval" value="<?=$rate_limit_interval?>" min="0" step="10" style="width: 4em;"> seconds.</p>
```

Give it 1em more....
![Bildschirmfoto zu 2022-02-26 14-29-13](https://user-images.githubusercontent.com/26622301/155845120-14bd2e81-c55c-4520-991b-b1d4060d47af.png)

Reply:
This is interesting, which browser (and possibly theme) is this? I tried with recent Firefox and Chrome and built-in themes on Linux and didn't see this.